layouts/partials/footer.html

```diff
@@ -38,11 +38,12 @@ <h2>{{ i18n "footer_support_us" }}</h2>
 
       <div class="footer-col  footer-col-2">
         <p class="text">{{ .Site.Params.description | safeHTML }}</p>
-        <div itemscope itemtype="http://schema.org/PostalAddress">
-          <span itemprop="streetAddress">1 Letterman Drive, Suite D4700,</span>
-          <span itemprop="addressLocality">San Francisco,</span>
+        <div itemscope itemtype="https://schema.org/PostalAddress">
```

Comment:
Could you point us to the doc validating that change?

Reply:
I must admit that I just assumed https would be as correct as http since the schema.org site supports HTTPS. I might be wrong. I can find a few examples of `https://schema.org/` usages but it seems all official examples use `http` instead, for instance https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/itemscope. I will revert the URL schema change from this pull request.

Reply:
As both urls points to the same page, and the MDN docs do have some examples with https, let's keep https 🙂

Reply:
Done

Reply:
> As both urls points to the same page, and the MDN docs do have some examples with https, let's keep https slightly_smiling_face

Oh, too late. I missed your reply. For another pull request in the future (TM) then... :)